analytical/templatetags/piwik.py

```diff
@@ -41,7 +41,19 @@
 
 VARIABLE_CODE = '_paq.push(["setCustomVariable", %(index)s, "%(name)s", "%(value)s", "%(scope)s"]);'  # noqa
 IDENTITY_CODE = '_paq.push(["setUserId", "%(userid)s"]);'
-DISABLE_COOKIES_CODE = '_paq.push([\'disableCookies\']);'
+DISABLE_COOKIES_CODE = "_paq.push(['disableCookies']);"
+ASK_FOR_CONSENT_CODE = "_paq.push(['requireConsent']);"
+FORGET_CONSENT_CODE = """
+document.getElementById("piwik_deny_consent").addEventListener("click",
+() => {
+  paq.push(["forgetConsentGiven"]);
+});
+"""
+REMEMBER_CONSENT_CODE = """ document.getElementById("piwik_give_consent").addEventListener("click",
+() => {
+  _paq.push(["setConsentGiven"]); _paq.push(["rememberConsentGiven"]);
```

Comment:
From the [docs you linked](https://developer.matomo.org/guides/tracking-javascript-guide#step-3-user-gives-consent), it seems only `_paq.push(['rememberConsentGiven']);` is required?

Reply:
You're right. Changed it in c3065fc0c96e762f064176f16554af440055cadc